apis/swagger.yml

```diff
@@ -266,13 +271,20 @@ paths:
         - name: id
           in: path
           required: true
-          description: "ID of piece"
+          description: "ID of task"
           type: string
+        - name: num
+          in: path
+          required: true
+          description: |
+            the range number of specific piece in the task.
+          type: integer
         - name: "PieceUpdateRequest"
           in: "body"
-          description: "request body which contains task update information"
+          description: |
+            request body which contains task update information.
           schema:
-            $ref: "#/definitions/PieceUpdateRequest"
+            $ref: "#/definitions/PieceInfo"
```

Comment:
Use `PieceUpdateRequest` instead of `PieceInfo`, the properties are:
```golang
// the downloader cid
srcCid string,
// the uploader cid
dstCid string,
// the peer task request status: START(700), RUNNING(701), FINISH(702)
status int,
// the peer task request result: FAIL(500), SUCCESS(501), INVALID(502), SEMISUC(503)
result int,
```